step2-02/README.md

```diff
@@ -2,38 +2,38 @@
 
 [Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
 
-UI Fabric is a Component Library that is developed to reflect the latest Microsoft design language. It is used in many Microsoft web applications and is developed in the open.
+[UI Fabric](https://developer.microsoft.com/en-us/fabric) is a component library that reflects the latest Microsoft design language. It is used in many Microsoft web applications and is [developed in the open](https://github.com/OfficeDev/office-ui-fabric-react).
 
 We'll talk about:
 
-- What makes it good
-- How to find it
-- How to use it
-- Laying out Apps with Stack
+- [What makes it good](#what-makes-it-good)
+- [How to find it](#how-to-find-it)
+- [How to use it](#how-to-use-it)
+- [Laying out apps with Stack](#layout-with-stack)
 
-# What Makes It Good
+## What Makes It Good
 
 - Fabric has been developed BOTH by developers and design engineers working together as a team
 - Most notable Microsoft web products use it
-- It is documented both from examples and generated from API (TypeScript) itself
+- It is documented both with examples and TypeScript API documentation
 - Components are highly customizable and themeable
 - Comprehensive library
-- Works with (aria) and augments (focus management) web accessibility standards
+- Works with assistive technologies and conforms to web accessibility standards for focus management
 - Fully funded and well managed - shield rotation and lots of automation work
-- Engineering is done in the open in github.com
+- Engineering is done in the open on GitHub
 - Engineering system is shared and re-usable by other teams
 
-# How to Find It
+## How to Find It
 
-github repo:
+GitHub repo:
 https://github.com/officedev/office-ui-fabric-react
 
-Documentation can be found here:
+Documentation:
 https://developer.microsoft.com/en-us/fabric/#/components
 
-# How to Use It
+## How to Use It
 
-## Importing a Component
+### Importing a Component
 
 ```jsx
 import { DefaultButton } from 'office-ui-fabric-react';
```

Comment:
This is kind of glossing over the fact that you have to do certain build setup to make an import like this work... But that might not be something we need to go into right away.

Reply:
webpack is going to be another session I'd be interested to do!